cypress/integration/cucumber/step_definitions/refapp-3.x/04-clinical-visit/clinical-visit.js

```diff
@@ -0,0 +1,139 @@
+import {Given, Before} from 'cypress-cucumber-preprocessor/steps';
+
+let identifier = null;
+let patient = null;
+
+
+Before({tags: '@clinical-visit'}, () => {
+    identifier = cy.generateIdentifier().then((generatedIdentifier) => {
```

Comment:
This initial `identifier = ` sets `identifier` to a Promise. I don't think that's what you want.

```suggestion
    cy.generateIdentifier().then((generatedIdentifier) => {
```